### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by EHale64**
Added a redirect to take the user back to the index page after sending the email.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ gem 'rails', '~> 6.1.1'
 
 gem 'bootsnap', '>= 1.4.4', require: false
 gem 'jbuilder', '~> 2.7'
+gem 'jsonapi-serializer', '~> 2.1'
```
Comment (nicorithner) [inline Gemfile:9]: We need to remove this one since we are no longer using it

File: app/serializers/contact_form_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class ContactFormSerializer
```
Comment (nicorithner) [inline app/serializers/contact_form_serializer.rb:1]: this serializer needs to be removed

**Review by nicorithner — CHANGES_REQUESTED** [review]
Can we remove the serializer gem and contact form serializer file?

**Review by npdarrington — COMMENTED** [review]
With the changes that Nico has already requested, looks good.

**Review by nicorithner — APPROVED** [review]

**Pull request merged**